internal/thelma/ops/sync/sync.go

```diff
@@ -33,13 +33,19 @@ type syncer struct {
 }
 
 // Sync a set of releases and return a status report indicating whether the release is healthy.
-func (s *syncer) Sync(releases []terra.Release, maxParallel int, options ...argocd.SyncOption) (map[terra.Release]*status.Status, error) {
+func (s *syncer) Sync(releases []terra.Release, maxParallel int, refreshOnly bool, options ...argocd.SyncOption) (map[terra.Release]*status.Status, error) {
 	var jobs []pool.Job
 
 	waitHealthyTimeout := s.extractWaitHealthy(options)
 
-	optionsNoWaitHealthy := withOption(options, func(options *argocd.SyncOptions) {
+	commonOptions := withOption(options, func(options *argocd.SyncOptions) {
 		options.WaitHealthy = false
+	}, func(options *argocd.SyncOptions) {
+		if refreshOnly {
```

Comment:
set this in sync_command.go so we don't have to add a new parameter to this function?

Reply:
That's what I did first, but then I can't set the text output at the bottom of this function, and it always says "1/1 services synced"

Reply:
IMO that's a worthwhile tradeoff but up to you!

Reply:
fair enough